### PR TITLE
Update owo translation

### PR DESCRIPTION
**Pull request by Noratrieb**
Some lines where not yet translated

### CODE REVIEW

File: kofta/public/locales/owo/translation.json [hunk 1]
```diff
@@ -232,16 +232,16 @@
...
-      "approachingGalaxy": "Approaching galaxy",
+      "fuelingRocket": "Fuewing rocket uwu",
+      "takingOff": "Tawking off xD",
+      "inSpace": "In space ʕʘ‿ʘʔ",
+      "approachingMoon": "Appwoaching moon  (❁´◡`❁)",
+      "lunarDoge": "Lunar doge (；ω；)",
+      "approachingSun": "Appwoaching sun >_<",
+      "solarDoge": "Solar doge :3",
+      "approachingGalaxy": "Appwoaching galaxy (◠‿◠✿)",
       "galacticDoge": "Galactic Doge",
```
Comment (jobveldhuis) [inline kofta/public/locales/owo/translation.json:243]: Are we sure this string doesn't need translating? 😄

**Review by jobveldhuis — APPROVED** [review]
I love the translations.

File: kofta/public/locales/owo/translation.json (hunk 1 above)
Comment (Noratrieb) [reply kofta/public/locales/owo/translation.json:243]: oh I forgot that line

File: kofta/public/locales/owo/translation.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "copied": "0w0 copied ^-^",
     "formattedIntlDate": "{{date, intlDate}}",
     "formattedIntlTime": "{{time, intlTime}}",
-    "requestPermissions": "Please note running DogeHouse without accessibility permissions may cause unwanted errors"
+    "requestPermissions": "<3 Pwease note ruwwing DogeHouse without accessabiwity pewmissiowns may cause ewwows ＞﹏＜"
   },
   "header": {
     "_comment": "Main Header UI Internationalization Strings",
```
Comment (drvladb) [inline kofta/public/locales/owo/translation.json:22]: Make sure that the Dogehouse name is consistent throughout the translation

File: kofta/public/locales/owo/translation.json (hunk 1 above)
Comment (amitojsingh366) [reply kofta/public/locales/owo/translation.json:243]: @Nilstrieb is this pr ready?

Comment (Noratrieb) [reply kofta/public/locales/owo/translation.json:243]: from my side, yes

**Review by samarmohan — APPROVED** [review]

**Pull request merged**